### DISCUSSION TITLE
How to change titleBar color

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/react-native-windows#10488] [Q&A] [answered]
![Screenshot 2022-09-01 141425](https://user-images.githubusercontent.com/65512279/187872891-358b4d3c-4090-4a2f-a2e5-e7310efaa5e6.png)

I need to change this background color

**Comment by chrisglein** [comment, marked as answer]
There's nothing specific that RNW surfaces to help with this. You'll use the UWP APIs to customize the titlebar: https://docs.microsoft.com/en-us/windows/apps/develop/title-bar?tabs=winui2

**Reply by ghost** [reply]
Can you gave an example, I can't see any App.xaml.cs in my project to customize
